code/modules/reagents/chemistry/recipes/drinks_reactions.dm

```diff
@@ -1063,3 +1063,11 @@
 	required_reagents = list("space_drugs" = 1, "sodawater" = 1, "grapejuice" = 1)
 	result_amount = 5
 	mix_sound = 'sound/goonstation/misc/drinkfizz.ogg'
+
+/datum/chemical_reaction/planet_cracker
+	name = "Planet Cracker"
+	id = "planet_cracker"
+	result = "planet_cracker"
+	required_reagents = list("fernet" = 10, "voxwine" = 10, "egg" = 2, "gold" = 5)
+	result_amount = 20
```

Comment:
```suggestion
	required_reagents = list("fernet" = 10, "voxwine" = 10, "egg" = 5, "gold" = 5)
	result_amount = 30
```
The drink already takes some weird stuff to make, I'd personally make this change so that it's easier to mix, but it's up to you